test/mpi/threads/perf/mt_rma_msgrate.c

```diff
@@ -34,7 +34,14 @@ enum RMA_OPs {
     OP_INVALID
 };
 
+enum TARGET_MODE {
+    TARGET_ACTIVE,
+    TARGET_PASSIVE,
+    TARGET_INVALID
```

Comment:
I would prefer more explicit names, e.g. `SYNC_MODE` and `USE_WIN_FENCE`, `USE_WIN_LOCK_ALL`

Reply:
Will do.